translations/cc-translations.js.it

```diff
@@ -21,7 +21,7 @@ function cc_js_t(s) {
  if (s == "We have updated the version of your license to the most recent one available.") { return "We have updated the version of your license to the most recent one available."; } 
  if (s == "Slovenia") { return "Slovenia"; } 
  if (s == "Guatemala") { return "Guatemala"; } 
- if (s == "The licensor permits others to copy, distribute and transmit the work. In return, licensees may not use the work for commercial purposes — unless they get the licensor's permission.") { return "Il <em>licenziante</em> acconsente che altri copino, distribuiscano, esibiscano ed eseguano l'opera. In cambio i <em>licenziatari</em> non possono usare l'<em>opera</em> per scopi commerciali &mdash; a meno che non abbiano ottenuto il permesso del <em>licenziante</em>."; } 
+ if (s == "The licensor permits others to copy, distribute and transmit the work. In return, licensees may not use the work for commercial purposes — unless they get the licensor's permission.") { return "Il <em>licenziante</em> acconsente che altri copinoo, distribuiscano, esibiscano ed eseguano l'opera. In cambio i <em>licenziatari</em> non possono usare l'<em>opera</em> per scopi commerciali &mdash; a meno che non abbiano ottenuto il permesso del <em>licenziante</em>."; } 
```

Comment:
Is `copinoo` a typo?

Reply:
yes sorry I changed for testing purposes without rolling back changes, please tell me if I shoud do a new pull request with this fix

Reply:
I'm so sorry for my delay to respond but you don't need to make a new pull request... you can add your commit in the same pull request.
Once you make the commit I'll accept the PR immediately.
Thank you so much for your contribution!